inst/shiny-server/server/upload.R

```diff
@@ -1,753 +1,757 @@
+  updateSelectInput(session, 'FV_NG.Heatmap.Funcid', choices = funcIds, selected = funcIds)
+  updateSelectInput(session, 'FV_NG.Heatmap.Dim', choices = DIMs, selected = DIMs)
+
+  updateSelectInput(session, 'RT_Stats.DSC.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'RT_Stats.DSC.Funcid', choices = funcIds, selected = funcIds)
+  updateSelectInput(session, 'RT_Stats.DSC.Dim', choices = DIMs, selected = DIMs)
+
+  updateSelectInput(session, 'FV_Stats.DSC.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FV_Stats.DSC.Funcid', choices = funcIds, selected = funcIds)
+  updateSelectInput(session, 'FV_Stats.DSC.Dim', choices = DIMs, selected = DIMs)
+
+  updateSelectInput(session, 'RT_Stats.Overview.ID', choices = IDs, selected = IDs)
+
+  updateSelectInput(session, 'FV_Stats.Glicko.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FV_Stats.Glicko.Funcid', choices = funcIds, selected = selected_f)
+  updateSelectInput(session, 'FV_Stats.Glicko.Dim', choices = DIMs, selected = selected_dim)
+
+  updateSelectInput(session, 'FV_Stats.Overview.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'RTSummary.Statistics.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'RTSummary.Overview.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FCESummary.Overview.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'RTSummary.Sample.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FV_PAR.Plot.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'RT_PAR.Plot.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FCESummary.Statistics.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FCESummary.Sample.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FV_PAR.Summary.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FV_PAR.Sample.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'RT_PAR.Summary.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'RT_PAR.Sample.ID', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'ERTPlot.Multi.Algs', choices = IDs, selected = selected_ID)
+  updateSelectInput(session, 'ERTPlot.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'ERTPlot.Aggr.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'ERTPlot.Aggr_Dim.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FCEPlot.Multi.Algs', choices = IDs, selected = selected_ID)
+  updateSelectInput(session, 'FCEPlot.Aggr.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FCEPlot.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FCEPDF.Bar.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FCEPDF.Hist.Algs', choices = IDs, selected = IDs)
+  updateSelectInput(session, 'FCEPDF.CDP.Algs', choices = IDs, selected = IDs)
```

Comment:
I would prefer having a check here to take the first two algorithms as selected by default, to limit confusion for the user when they open the plot